lib/snowflex_id/table.ex

```diff
@@ -1,10 +1,11 @@
-defmodule SnowflexTable do
-  @moduledoc """
-  Keeps track of Snowflex nodes and sequential number with ETS.
-  """
+defmodule SnowflexId.Table do
```

Comment:
Mixed feeling about this namespace convention. In a not so short file Table would not be very helpful and as-is wouldn't need an "as" on the alias to specify what Table.